src/main/resources/templates/apidocumentation/apiDocumentationForTag.ftl

```diff
@@ -42,7 +42,8 @@
         No authentication is required for any endpoints.
       </div>
 
-      <@govukAccordion.accordion accordionId="elg-api-operations">
+      <#global tagNameSanitised=tagName?replace(" ", "-")>
```

Comment:
This can probably be a #local rather than a #global

Reply:
@matteason You can only use #local inside a function or macro